brooklyn-server/camp/camp-brooklyn/src/test/java/org/apache/brooklyn/camp/brooklyn/ExternalConfigYamlTest.java

```diff
@@ -108,6 +110,34 @@ public void testExternalisedLocationConfigReferencedFromYaml() throws Exception
         assertEquals(Iterables.getOnlyElement( app.getLocations() ).config().get(MY_CONFIG_KEY), "myval");
     }
 
+    @Test
+    public void testExternalisedCatalogConfigReferencedFromYaml() throws Exception {
+
+        String yaml = Joiner.on("\n").join(
+                "brooklyn.catalog:",
+                "    id: osgi.test",
+                "    itemType: template",
+                "    version: 1.3",
+                "    description: CentOS 6.6 With GUI - 1.3",
+                "    displayName: CentOS 6.6",
+                "    iconUrl: classpath:///centos.png",
+                "    brooklyn.libraries:",
+                "    - $brooklyn:external(\"myprovider\", \"osgikey\")",
```

Comment:
I see a problem using the name/version in test, because we need a real one that match the url and if they are changed in future, someone would wonder why the test fails.

There is one more test without external config to verify that the behaviour doesn't change then.

If you think it's not that bad to add matching for now name/version, no problem to add them.